.travis/install.sh

```diff
@@ -33,7 +33,7 @@ if grep -E '(unit-tests|functional-tests)' <<< "${args[@]}"; then
 fi
 
 if grep 'unit-tests' <<< "${args[@]}"; then
-    pip install pycodestyle pytest
+    pip install pycodestyle pytest mock testfixtures
```

Comment:
You can now use `unittest.mock` from standard library.

Reply:
Done.